commands/gban.js

```diff
@@ -23,6 +23,12 @@ exports.run = async (client, message, args) => {
 
     const potentialGuilds = client.guilds.filter(g=>g.me.hasPermission('BAN_MEMBERS'))
 
+    let completemessage;
+    let clientmember = await message.guild.members.get(client.user.id)
+
+    if (message.channel.permissionsFor(clientmember).has('SEND_MESSAGES'))
```

Comment:
Also unnecessary, for now, let's assume the bot can write in that channel. I'll add a check on every command before trying trying to execute it.

Reply:
No it isn't. Why assume things that might not be true.
Besides, why add a check that will not let the command run if the only thing that needs to be changed is just make it not send a message?

Reply:
Add it on the message event before trying to run any command and just return if there's no send message/embed links permission.

Reply:
No, cause there's no need for that.
Those commands could be easily redirected to DM.